src/figuresClasses.ts

```diff
@@ -1,11 +1,73 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
There's no need to assign the constructor parameters to class properties manually since TypeScript does this automatically when you use an access modifier like 'public' in the constructor parameter list.